### ISSUE TITLE
Question: Desktop platform?

### ISSUE THREAD

**Issue by eliezedeck** [Larpon/QtFirebase#61]
I know the official C++ SDK only targets iOS + Android only with a desktop stubs.

But does this library makes necessary steps to make some features to work on desktops? Like, for example, using the REST API for the Realtime database.

**Comment by Larpon** [comment]
@eliezedeck - hi! Thanks for checking out the project!

No - we primarily focus solely on keeping it as close (and bug free) to the C++ SDK as possible.

However I've read, several places, that Google are considering support for desktop builds. But so far they've been very reluctant on the subject.

Should you have an implementation of the REST API at hand - I'll gladly think about taking it in as a PR. But our main concern is the C++ SDK - which is, as you correctly state, Android and iOS only at the moment.  :)

**Comment by eliezedeck** [comment]
Thanks for the clarification.

So, it will be a duplicate effort if I tried to use a REST API, and then the Firebase team will announce an official desktop support. I've also looked at the current SDK, I didn't really try to use it yet, but seeing the **binary size**, it might be possible that there is already work in progress in that area. I mean, for a stub implementation, it surprisingly _big binaries_ in there.

I'll try to reach the team and learn about their current progress on that desktop area.

Closing.
